sys/include/net/nanocoap.h

```diff
@@ -265,6 +286,7 @@ typedef ssize_t (*coap_handler_t)(coap_pkt_t *pkt, uint8_t *buf, size_t len, voi
  */
 typedef struct {
     const char *path;               /**< URI path of resource               */
+    unsigned path_opt;              /**< URI path options                   */
```

Comment:
We should use a free bit of the methods field. For the example alone, this wastes 20 bytes RAM otherwise, on ARM.

I suggest calling it COAP_MATCH_SUBTREE or something similar.

Reply:
Changed to use the methods field